Sources/TuistGenerator/Generator/ProjectFileElements.swift

```diff
@@ -304,11 +304,16 @@ class ProjectFileElements {
 
         // Add the file element
         if isLocalized(path: absolutePath) {
-            addVariantGroup(from: from,
-                            absolutePath: absolutePath,
-                            relativePath: relativePath,
-                            toGroup: toGroup,
-                            pbxproj: pbxproj)
+            // Localized container (e.g. /path/to/en.lproj) we don't add it directly
+            // an element will get added once the next path component is evaluated
+            //
+            // note: assumption here is a path to a nested resource is always provided
```

Comment:
I'd show a warning if that assumption is not met.

Reply:
I was testing further - looks like we're good on this front (almost)

in the event a resource in the manifest is defined as `resources: ["Resouces/en.lproj"]` it will get filtered out as it's not a file per se - and a warning is displayed:

```
⚠️  Warning: No files found at: Resources/en.lproj
```

The warning could do with an improvement to clarify the situation further e.g.

```
⚠️  Warning: Resources/en.lproj is a directory, try using Resources/en.lproj/** to search for files
```

Side Note: It's looking like there are few cases now that could warrant a manifest linter - as there's additional metadata there we could use to display more useful warnings and errors too. We can re-examine that if we get more of these style warnings 🙂

Reply:
Awesome! Thanks for clarifying @kwridan